spec/unit/models/user_filters/user_filter_builder_spec.rb

```diff
@@ -0,0 +1,100 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+describe GoodData::UserFilterBuilder do
+  describe '.execute_mufs' do
+    let(:login) { 'rubydev+admin@gooddata.com' }
+    let(:full_definition_filter) do
+      {
+        :login=> login,
+        :filters=>[{
+          :label=>label_id,
+          :over=>nil,
+          :to=>nil,
+          :values=>["Washington"]}]
```

Comment:
Style/MultilineHashBraceLayout: Closing hash brace must be on the line after the last hash element when opening brace is on a separate line from the first hash element.